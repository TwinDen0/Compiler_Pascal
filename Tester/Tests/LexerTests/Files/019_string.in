#255
#256
#65535
#65536