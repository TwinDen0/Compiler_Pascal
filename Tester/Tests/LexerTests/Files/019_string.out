(1,4) Fatal: Syntax error, ";" expected
